lib/index.coffee

```diff
@@ -21,6 +22,9 @@ lint = (editor) ->
   convertOldConfig()
   command = atom.config.get(COMMAND_CONFIG_KEY).split(/\s+/).filter((i) -> i)
     .concat(DEFAULT_ARGS, filePath = editor.getPath())
+  if atom.config.get(DISABLE_CONFIG_KEY) is true
+    config = helpers.find(filePath, '.rubocop.yml')
+    return [] if config is null or not atom.project.rootDirectories.some (p) -> config.startsWith(p.path)
```

Comment:
What's the reasoning behind this second check? Are you trying to limit it to results within the current project?